lib/specinfra/command/module/service/delegator.rb

```diff
@@ -0,0 +1,20 @@
+module Specinfra
+  module Command
+    module Module
+      module Service
+        module Delegator
+          def def_delegator_service_under(under)
```

Comment:
I intentionally did not validate possible providers here because it restricts to append extra providers from gem.